ui/src/app/components/runtime-configuration-panel/customize-panel-footer.tsx

```diff
@@ -87,12 +91,12 @@ export const CustomizePanelFooter = ({
       <LinkButton
         style={{
           ...styles.deleteLink,
-          ...(disableControls || !runtimeExists
+          ...(disableControls || !canDeleteRuntime(currentRuntime?.status)
```

Comment:
there are many different ways to refer to runtimes and their statuses. "exists" includes an isVisible() check which excludes the ERROR state. This is appropriate in many cases, but not for deletion.